packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -198,7 +198,7 @@ export class OdspSummaryUploadManager {
             cloneDeep(tree),
             blobTreeDedupCachesLatest,
             ".app",
-            true,
+            false /* Stop handle expansion due to missing unreferenced flag in summary returned from server */,
```

Comment:
It would be great to add a reference to an issue or some other description that tells why and potential next steps.
I.e. next developer reading code should be able to reason on whys and how he/she can improve it further.

Reply:
Added.